docs/modules/configuring/con-configuring-cluster-connections.adoc

```diff
@@ -0,0 +1,79 @@
+[id='con-configuring-cluster-connections-{context}']
+= Securing connections to target clusters
+
+[role="_abstract"]
+To secure a virtual cluster connection to a target cluster, configure TLS on the virtual cluster.
+The target cluster must already be configured to use TLS. 
+
+Specify TLS for the virtual cluster configuration using `targetCluster.tls` properties 
+
+Use an empty object (`{}`) to inherit trust from the OpenShift platform. 
```

Comment:
it could be running on bare metal, kubernetes or OpenShift.

Reply:
Can we be more generic here?  Something like: "Use an empty object (`{}`) to inherit trust from the underlying platform on which the cluster is running."  (Trying to keep it suitable for downstream)

Reply:
Yes, that wording makes sense.